lib/ansible/playbook/role_include.py

```diff
@@ -53,6 +53,7 @@ class IncludeRole(TaskInclude):
     # private as this is a 'module options' vs a task property
     _allow_duplicates = FieldAttribute(isa='bool', default=True, private=True)
     _public = FieldAttribute(isa='bool', default=False, private=True)
+    _rolespec_validate = FieldAttribute(isa='bool', default=True, private=True)
```

Comment:
shoudl not be private, though it really does nothing at this point

Reply:
removed it in next commit